packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -92,63 +56,64 @@ describe('packages/mongo-nav', () => {
     });
 
     test('the organization and project navs are rendered', () => {
-      expect(expectedElements.orgNav).toBeInTheDocument();
-      expect(expectedElements.projectNav).toBeInTheDocument();
+      expect(queryByTestId('organization-nav')).not.toBeNull();
+      expect(queryByTestId('project-nav')).not.toBeNull();
     });
 
     test('current orgId is set based on data returned from fetch', () => {
-      expect((expectedElements.billing as HTMLAnchorElement).href).toBe(
+      expect((getByTestId('org-nav-billing') as HTMLAnchorElement).href).toBe(
```

Comment:
I'm curious if we make a page object that collects all of these string id's for a clearer representation of the parts of the nav.

```jsx
const MongoNavElements = {
   OrgNav: {
       Billing: 'org-nav-billing'
   }
}
```

and etc?

I'm a little on the fence as it seems like adding indirection, but we are repeating ourselves a lot. Also having a canonical list of all id references seems pleasant as well.

Reply:
would we want something like this to also be used in the components themselves? e.g. `data-testid={MongoNavElements.OrgNav.Billing}`. any thoughts on where to locate these, e.g. separate file, same file as component

Reply:
I don't feel as strongly about collecting all source `data-testid` values as they're only used once.

The obvious benefit to collect them is it becomes easily reusable for tests.

For MongoNav it may be useful to consolidate them all as it's so massive.

Reply:
I think I want to hold off on his for now, just to reduce the scope of the change

Reply:
WFM